src/Features/VisualBasic/Portable/Completion/KeywordRecommenders/Expressions/TrueFalseKeywordRecommender.vb

```diff
@@ -12,12 +13,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.KeywordRecommenders.Expr
         Inherits AbstractKeywordRecommender
 
         Protected Overrides Function RecommendKeywords(context As VisualBasicSyntaxContext, cancellationToken As CancellationToken) As IEnumerable(Of RecommendedKeyword)
+            Dim preselect As Boolean = ShouldPreselect(context, cancellationToken)
+
             If context.IsAnyExpressionContext Then
-                Return {New RecommendedKeyword("True", VBFeaturesResources.TrueKeywordToolTip),
-                        New RecommendedKeyword("False", VBFeaturesResources.FalseKeywordToolTip)}
+                Return {New RecommendedKeyword("True", VBFeaturesResources.TrueKeywordToolTip, shouldPreselect:=preselect),
+                        New RecommendedKeyword("False", VBFeaturesResources.FalseKeywordToolTip, shouldPreselect:=preselect)}
             End If
 
             Return SpecializedCollections.EmptyEnumerable(Of RecommendedKeyword)()
         End Function
+
+        Private Function ShouldPreselect(context As VisualBasicSyntaxContext, cancellationToken As CancellationToken) As Boolean
+            Dim documentId = context?.Workspace.CurrentSolution.GetDocumentIdsWithFilePath(context.SyntaxTree.FilePath).FirstOrDefault()
```

Comment:
Is this safe to call? I would expect that CurrentSolution may not be in sync with the document and strange things might happen if you're typing fast enough. Looking at VisualBasicSyntaxContext, the constructor accepts a SemanticModel but throws it away. We might be able to get semantics here properly (although I admit the type might be need renaming then.)